src/tests/Microsoft.PowerFx.Core.Tests/FunctionTests.cs

```diff
@@ -36,6 +38,42 @@ public void TestAllBuiltinFunctionsHaveParameterDescriptions()
             }
         }
 
+        [Fact]
+        public void FunctionCategoriesLocaleTest()
+        {
+            var texlFunctionsLibrary = BuiltinFunctionsCore.BuiltinFunctionsLibrary;
+            var locale_enus = CultureInfo.CreateSpecificCulture("en-US");
+            var locale_ptbr = CultureInfo.CreateSpecificCulture("pt-BR");
+
+            foreach (var func in texlFunctionsLibrary)
+            {
+                var info = new FunctionInfo(func);
```

Comment:
I don't think we need to call on every function in the library - lots of overlap. 
We can just call on one specific one - and assert the actual contents. 

Can we move this to FunctionInfoTests.cs ?